frontend-react/src/hooks/api/messages/UseMessageTesting.ts

```diff
@@ -0,0 +1,34 @@
+import { useSuspenseQuery } from "@tanstack/react-query";
+import { useCallback } from "react";
+import { reportsEndpoints, RSMessage } from "../../../config/endpoints/settings";
+import useSessionContext from "../../../contexts/Session/useSessionContext";
+import { Organizations } from "../../UseAdminSafeOrganizationName/UseAdminSafeOrganizationName";
+
+const { testing } = reportsEndpoints;
+
+const useReportTesting = () => {
```

Comment:
Name does not convey what it's for specifically. Unless this mimics what the test message entity is named in backend code (oh boy if it does...) it should be something similar to `useTestMessages`.